README.md

```diff
@@ -486,6 +486,7 @@ Hurray! You successfully made a contribution! 🎉
   - [Russ Perry](https://github.com/rperry99)
   - [Rutik Bhoyar](https://github.com/Rutikab12)
   - [Rwitesh Bera](https://github.com/rwiteshbera)
+  - [Rahul Kumar Roy](https://github.com/mRahulRoy)
```

Comment:
Please add your name alphabetically (Add your name in 3rd position on Line on `459`)